www/components/Sidebar/Sidebar.tsx

```diff
@@ -21,14 +21,19 @@ const Container = styled(Box)`
   height: 100%;
 `;
 
+const StyledLink = styled(Link)`
+  color: ${(props) => props.theme.colors.dark[800]};
+  text-decoration: none;
+`;
+
```

Comment:
seems unnecessary for this PR :stuck_out_tongue:

Reply:
Negative; without this, the link gets the browser-default blue/purple styling. (This didn't happen before because we used the Next.JS `Link`, which takes care of this; now we use the `react-router-dom` `Link`, which doesn't)